policy/policy.go

```diff
@@ -110,8 +110,14 @@ func WaitUntilDone(resolver PolicyResolver, entity string, scoringMrn string, ti
 
 // RefreshLocalAssetFilters looks through the local policy asset filters and rolls them up
 func (p *Policy) RefreshLocalAssetFilters() {
-	p.Filters = &explorer.Filters{
-		Items: map[string]*explorer.Mquery{},
+	if p.Filters == nil {
```

Comment:
Same as above comment, the job of this function was to create p.Filters and clear it out.